servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/MessageCodec.java

```diff
@@ -19,10 +19,21 @@
 import io.servicetalk.buffer.api.BufferAllocator;
 
 /**
- * Codec used to encode and decode gRPC messages.
- * This instance is shared across all requests/responses therefore it must provide thread safety semantics.
+ * Codec used to encode and decode {@link Buffer} content.
+ * This instance is shared therefore it must provide thread safety semantics.
  */
-public interface GrpcMessageCodec {
+public interface MessageCodec {
```

Comment:
Just curious, why `Grpc` prefix was removed? :)

Reply:
The gRPC PR removes them completely, https://github.com/apple/servicetalk/pull/1198
Should keep the changes there, or you would rather bring them here?

Reply:
Doesn't matter, let's handle it in $1198. Thanks for clarifying.